### PR TITLE
HOMS-220 Improve README

### PR DESCRIPTION
**Pull request by dolfinus**
Cleaning up development installation instruction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,103 +65,60 @@ The prefered way to install HOMS is to use docker
...
-
-  ```
-  sources:
-    bpmanagementsystem:
-      type: static/activiti
+  find config -name '*.sample' | xargs -I{} sh -c 'cp $1 ${1%.*}' -- {}
   ```
 
-9. Run docker-compose:
+6. Skip this step
```
Comment (mxsmirnoff) [inline README.md]: No step 5

File: README.md [hunk 2]
```diff
@@ -65,103 +65,60 @@ The prefered way to install HOMS is to use docker
...
 
-8. Run `docker-compose`:
+7. Run `docker-compose`:
 
   ```
   docker-compose up -d
   ```
 
-Login at [http://localhost:3000](http://localhost:3000) with *`user@example.com`*/*`changeme`*.
+8. Go to [http://127.0.0.1:9000](Minio) with *`$MINIO_ACCESS_KEY`*/*`$MINIO_SECRET_KEY`* and create there bucket with name equal to `minio.bucket_name` value from `/etc/hydra/homs/hbw.yml` config file.
```
[the remark this answers is not in the document]
Comment (dolfinus) [reply README.md]: Fixed

**Comment by sugarfree1** [comment]
Squash commits into one commit and give it proper name.

**Comment by dolfinus** [comment]
Done

File: README.md [hunk 3]
```diff
@@ -65,103 +65,60 @@ The prefered way to install HOMS is to use docker
...
+
 
 #### In development
 
 If you don't want to use Oracle as source for your HOMS instance:
 
 ##### Without Oracle Instant Client
 
-1. Clone HOMS git repository:
+Follow the same steps as for [In production installation](#in-production), but with certain changes:
```
Comment (sugarfree1) [inline README.md]: Add new line after this line.

Comment (dolfinus) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -65,103 +65,61 @@ The prefered way to install HOMS is to use docker
...
 
-8. Run `docker-compose`:
+7. Run `docker-compose`:
 
   ```
   docker-compose up -d
   ```
 
-Login at [http://localhost:3000](http://localhost:3000) with *`user@example.com`*/*`changeme`*.
+8. Go to [Minio](http://127.0.0.1:9000) with *`$MINIO_ACCESS_KEY`*/*`$MINIO_SECRET_KEY`* and create there bucket with name equal to `minio.bucket_name` value from `/etc/hydra/homs/hbw.yml` config file.
```
[the remark this answers is not in the document]
Comment (dolfinus) [reply README.md]: Fixed

**Pull request merged**